.github/workflows/image.yaml

```
@@ -0,0 +1,128 @@
+# Build and push images to :
+# GitHub Container Registry (ghcr.io)
+# Red Hat Container Registry (quay.io)
+name: "Build Docker Images"
+
+on:
+  create: # Branch (so that the Docker image tags exist for a new branch when pushed)
+          # even if nothing on 'paths' below is changed.
+    branches:
+      - '**'
+
+  push:
+    paths:
+      - "**"
+      - ".github/workflows/image.yaml"
+
+    branches: # Only on paths above
+      - '**'
+
+    tags: # Always, regardless of paths above
+      - '*'
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write
+  security-events: write
+
+jobs:
+  jupyterlab:
```

```suggestion
  build-images:
```
best to use descriptive names